test/crawl-test.js

```diff
@@ -0,0 +1,90 @@
+var promise = require('chai').expect
+var assert = require('chai').assert
+var expect = require('chai').expect
+var _ = require('lodash');
+var Crawler = require('../src/crawler.js').Crawler;
+
+describe('Crawler', function() {
+  describe('#constructor', function() {
+    // max requests
+    it('Shouldn\'t return an error when given valid max requests', function() {
+      var crawler = new Crawler(100, console);
+    });
+    it('Shouldn\'t return an error when given undefined max requests', function() {
+      var crawler = new Crawler(undefined, console);
+    });
+    it('Should return an error when given non numeric max requests', function() {
+      var maxRequests = "lwarc";
+      expect(function() {new Crawler(maxRequests)}).to.throw(Error);
+    });
+
+    // logger
+    it('Shouldn\'t return an error when given valid logger', function() {
+      var crawler = new Crawler(100, console);
+    });
+    it('Shouldn\'t return an error when given undefined logger', function() {
+      var crawler = new Crawler(100, undefined);
```

Comment:
Just to be extra sure, this should be ``new Crawler(100)``, as in drop the second argument entirely.